shared/package.json

```diff
@@ -23,7 +23,7 @@
     "build-dev": "yarn run _helper build-dev",
     "build-emoji": "yarn run _node common-adapters/markdown/generate-emoji-parser.js",
     "build-prod": "yarn run _helper build-prod",
-    "build-actions": "yarn run _node desktop/yarn-helper/action-creator-creator",
+    "build-actions": "yarn run _node --extensions \".tsx\" desktop/yarn-helper/action-creator-creator.tsx",
```

Comment:
i don't think its necessarily required converting all these files over unless its worth it

Reply:
It was easy so why not /shrug. but good to know for future cases